libs/ui/packages/shared/src/assorted/Icon/Providers.ts

```diff
@@ -0,0 +1,56 @@
+import { CDN_URL } from "@ledgerhq/ui-shared";
+
+export const ALL_PROVIDERS = [
+  "aave",
+  "baanx",
+  "banxa",
+  "binance",
+  "bitrefill",
+  "btcdirect",
+  "changelly",
+  "cic",
+  "coingecko",
+  "coinify",
+  "compound",
+  "deversifi",
+  "ftx",
+  "ftxus",
+  "kiln",
+  "lido",
+  "loopipay",
+  "mercuryo",
+  "moonpay",
+  "oneinch",
+  "opensea",
+  "pancakeswap",
+  "paraswap",
+  "poap",
+  "rainbowme",
+  "ramp",
+  "rarible",
+  "simplex",
+  "sushiswap",
+  "transak",
+  "wyre",
+  "zerion",
+] as const;
+
+export type Provider = (typeof ALL_PROVIDERS)[number];
+
+export const isProviderIconName = (name: string): name is Provider =>
+  ALL_PROVIDERS.some((item) => item === name);
```

Comment:
I am not sure about this check...
For instance, ETH stake banner is controlled by a FF, if Product change the FF data, the banner will change the appearance.
If we need to add a new provider to list, we will need to plan the change with 3 weeks in advance (LLM and LLD releases)
Can we avoid this check?
```
{"enabled":true,"params":{"listProvider":[{"id":"kiln_pooling","liveAppId":"kiln","supportLink":"https://www.kiln.fi","icon":"Kiln:provider","queryParams":{"focus":"pooled"}},{"id":"kiln","liveAppId":"kiln","supportLink":"https://www.kiln.fi","icon":"Kiln:provider","queryParams":{"focus":"dedicated"}}]}}
```

Reply:
List of supported providers, icon name validation has been removed to keep this more flexible